etc/kayobe/kolla.yml

```diff
@@ -75,7 +75,7 @@ kolla_docker_namespace: rse-cambridge
 #kolla_docker_registry_password:
 
 # Kolla OpenStack release version. This should be a Docker image tag.
-kolla_openstack_release: 7.0.1.9
+kolla_openstack_release: 7.0.1.10
```

Comment:
Are you sure about this one, reverts the seed upgrade

Reply:
This is fine, I think, as we're incrementing the Kolla release here.

Reply:
ah, oops, read that the wrong way around.